lib/events/embeddable.js

```diff
@@ -0,0 +1,91 @@
+import stringify from 'json-stable-stringify'
+import validate from '../validate'
+
+const EVENT_TYPES = ['rendered', 'seen', 'clicked']
+const CONTENT_TYPES = ['product', 'brand', 'ask_an_owner']
+const HIT_TYPES = ['impression', 'non_impression', 'miss']
+const IMPLEMENTATIONS = ['standalone', 'tabbed']
+
+function validationsForEventType(/* eventType */) {
+  return {
+    eventType: {
+      presence: true,
+      inclusion: EVENT_TYPES,
+    },
+    contentType: {
+      presence: true,
+      inclusion: CONTENT_TYPES,
+    },
+    hitType: {
+      presence: false,
+      inclusion: HIT_TYPES,
+    },
+    implementation: {
+      presence: false,
```

Comment:
presence should be true here as we always want to know what is the implementation

Reply:
sure